normandy/health/api/views.py

```diff
@@ -28,3 +32,46 @@ def version(request):
         'source': 'https://github.com/mozilla/normandy',
         'commit': get_commit(),
     })
+
+
+@api_view(['GET'])
+def health(request):
+    include_deployment_checks = not settings.DEBUG
+    all_checks = checks_registry.get_checks(include_deployment_checks)
+
+    details = {check.__name__: health_check_detail(check) for check in all_checks}
```

Comment:
Is grouping useful? If not it might simplify this view, and all the checks should have descriptive messages anyway.